tools/prep/prepare_space_xml.py

```diff
@@ -118,6 +118,19 @@ def filter_model_validation_benchmarks(space, select_benchmarks):
         if (not is_model_validation_benchmark(b)):
             space.remove(b)
 
```

Comment:
We already have a `is_competitive`: https://github.com/SMT-COMP/smt-comp/blob/master/tools/prep/prepare_space_xml.py#L193
This is used here to not add solvers to the space if divisions is not competitive: https://github.com/SMT-COMP/smt-comp/blob/master/tools/prep/prepare_space_xml.py#L234
And in turn, a space without solvers is then removed here:  https://github.com/SMT-COMP/smt-comp/blob/master/tools/prep/prepare_space_xml.py#L234

I'll add more comments to document this better.